roles/rac-db-setup/tasks/rac-db-install.yml

```diff
@@ -137,6 +122,21 @@
       - "{{ install_rac_db }}"
   tags: rac-db,rac-db-install
 
+- name: rac-db-install | Update DB OPatch
```

Comment:
IIUC, the task block `rac-db-install | Update DB OPatch` has been moved in this PR after the task `rac-db-install | Set installer command`.

Upon eyeballing the pre & post blocks, it seems the content is same. Does the new ordering have any functionality effect?

Reply:
Indeed the order does matter.

Unlike the GI install, the DB install is out of place, and runInstaller actually clobbers existing content in the ORACLE_HOME, including the OPatch executable. Therefore we need to update OPatch _after_ the install has completed, but before invoking it to apply one-off patches